Cable/Parsers/cableUtils.h

```diff
@@ -36,7 +36,7 @@
 
 // Visual C++ for-loop scoping hack.  We can use this inside the CABLE
 // application without interfering with user code.
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && _MSC_VER < 1700
 #ifndef for
 #define for if(false) {} else for
 #endif
```

Comment:
This was back from the VS 6 days (`_MSC_VER < 1300`).  We can drop the whole block.